src/common/constant-time.h

```diff
@@ -131,4 +131,20 @@ constant_time_eq_i(int a, int b)
 	return constant_time_eq((unsigned int)a, (unsigned int)b);
 }
 
+/* Signed comparisons */
+
+static constant_inline unsigned int
+constant_time_ge_signed(int a, int b)
```

Comment:
Looks good ✨  Let's just move this below `constant_time_ge` together with the other comparison functions. I would also suggest renaming it to `constant_time_ge_i` in consistence with `constant_time_eq` and `constant_time_eq_i`.
```suggestion
constant_time_ge_i(int a, int b)
```

Reply:
good idea to rename 👍